app/javascript/controllers/content_list_controller.js

```diff
@@ -2,10 +2,17 @@ import { Controller } from "@hotwired/stimulus"
 import Thumbnail from 'src/modules/thumbnail'
 
 export default class extends Controller {
-  // This listens for a thumbnails-found event. Then it draws each on the page.
-  draw(evt) {
+  // This listens for a thumbnails-found event. Then it draws the media thumbnails in the contents panel
+  drawThumbnails(evt) {
     const thumbnails = evt.detail.
       map((thumbnailData, index) => new Thumbnail(thumbnailData).build(index))
     this.element.innerHTML = thumbnails.join('')
   }
+
+  // show a different PDF when the user selects it in the list
+  showPdf(evt) {
```

Comment:
This behavior is already in `ThumbnailController#activate`. I don't think we should make this special solution for PDFs.

Reply:
In either case, I think the action that handles switching the viewer should be in the same controller for both.  It's confusing to have the "change item" be in one place for media and another place for PDF.

Reply:
Makes sense.  I moved it to the content-list controller and named it similarly to the new `showPdf` method for consistency.